crates/sui-benchmark/src/lib.rs

```diff
@@ -264,6 +264,9 @@ impl ValidatorProxy for LocalValidatorAggregatorProxy {
     }
 
     async fn execute_transaction(&self, tx: Transaction) -> anyhow::Result<ExecutionEffects> {
+        if std::env::var("BENCH_MODE").is_ok() {
```

Comment:
So does this require `BENCH_MODE` to be set outside of simtests? Maybe instead we can run `execute_transaction()` only in tests and simtests?

Reply:
Yes, this change explicitly wants the user to set `BENCH_MODE ` if they want to use `execute_bench_transaction()` which does not handle epoch changes like the quorum driver. So, the default behavior is favoring correctness (without the env variable set) and allows user to switch to more performant code path if they want to.

Reply:
For instance I encountered this locally that stress client started failing after epoch change when i used a small epoch duration

Reply:
This is good point. I think this gets into the debate for whether stress client is intended for benchmarking or generating transactions load. If we require users to set `BENCH_MODE`, I think @asonnino should be aware so he can add the flag when running tests. Also we need to add the flag when running stress client in `sui-operations`.